### ISSUE TITLE
Version 1.3.5 not packages and available at PyPI

### ISSUE THREAD

**Issue by tuergeist** [mailjet/mailjet-apiv3-python#96]
PyPI still lists version 1.3.4 from 2020. However, there seem to be issues with that version (API changes?)

Please upload your Version 1.3.5 to PyPi to make everything work properly.

**Comment by sowinski** [comment]
Hi, has Mailjet gone bankrupt? Does anyone know if Mailjet is still active?

**Comment by skupriienko** [comment]
Yes, Mailjet is still maintained. The `master` branch has been updated recently, and a new release (1.4.0?) is preparing.
Before that happens you can clone the repo and run:
`make install`
or
`make dev`
